Translation corrections (#454)
07th-mod/higurashi-console-arcs: Update/ura_015.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>速水</color>", NULL, "<color=#f5e6d3>Hayami</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s14/00/vt4bb_hayami1035", 256, TRUE);
	OutputLine(NULL, "今なら弁護士を雇う権利と、黙秘する権利を特別に認めてやるぞ～』",

> 			NULL, "If you do, I can grant you the right to a lawyer and the right to remain silent on the house~.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "If you do, I can grant you the right to a lawyer and the right to remain silent, on the house~.\"", Line_Normal);
```

//rまともにやり合って、勝算などない…あるわけがない！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "まともにやり合って、勝算などない…あるわけがない！",

> 			NULL, "They had no chance of winning a fair fight... It was impossible!", Line_Normal);

Reviewer: ```suggestion
			NULL, "They had no chance in a fair fight... It was impossible!", Line_Normal);
```

//r常に眉間にしわを寄せ気味だった岡部の顔に、…まるで憑きものが取れたようなすっきりとした薄い笑みが浮かぶ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "常に眉間にしわを寄せ気味だった岡部の顔に、…まるで憑きものが取れたようなすっきりとした薄い笑みが浮かぶ。",

> 			NULL, "Okabe's face had a thoroughly furrowed brow up until a moment ago... but now he had a refreshing smile on his face, as though freed from an evil spirit.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Okabe's face had a thoroughly furrowed brow up until a moment ago... but now he had a refreshing smile on his face, as though freed from whatever possessed him.", Line_Normal);
```

			NULL, "\"...Aha!", Line_WaitForInput);
	ModPlayVoiceLS(4, 0, "ps3/s14/00/vt4bb_manc1003", 256, TRUE);
	OutputLine(NULL, "　ありました、社長！」",

> 			NULL, " Mr. President, we found it!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Boss, we found it!\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>速水</color>", NULL, "<color=#f5e6d3>Hayami</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s14/00/vt4bb_hayami1067", 256, TRUE);
	OutputLine(NULL, "「…あのプライドの塊の前野が、是非にもと言って頭を下げてきたんだ。",

> 			NULL, "\"...That prideful Maeno bowed his head and begged us to do it.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"...As prideful as Maeno is, he bowed his head and begged us to do it.", Line_WaitForInput);
```

//rその覚悟とともに、花田は輸送機に向かって再び駆け出す。迷っている時間は、もうほとんど残されていなかった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "その覚悟とともに、花田は輸送機に向かって再び駆け出す。迷っている時間は、もうほとんど残されていなかった。",

> 			NULL, "With that resolution in his heart, Hanada resumed rushing toward the plane. He had no time to waste hesitating.", Line_Normal);

Reviewer: ```suggestion
			NULL, "With that resolution in his heart, Hanada resumed rushing toward the plane. He had no time to waste being hesitant.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>浅見</color>", NULL, "<color=#f5e6d3>Asami</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s14/00/vt4bb_asami1051", 256, TRUE);
	OutputLine(NULL, "『絶対…とはいえません！",

> 			NULL, "\"I can't say... it's impossible! ", GetGlobalFlag(GLinemodeSp));

Reviewer: ```suggestion
			NULL, "\"I... can't say it's impossible! ", GetGlobalFlag(GLinemodeSp));
```

			NULL, " If that plane makes it up in the air, we're done for!", Line_WaitForInput);
	ModPlayVoiceLS(4, 0, "ps3/s14/00/vt4bb_yoshika1042", 256, TRUE);
	OutputLine(NULL, "　口で言っても止められないなら車でもぶつけて、",

> 			NULL, " If nothing we say can stop it, then hit it with a car or some...", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " If nothing we say can stop it, then hit it with a car or someth...", Line_WaitForInput);
```

//r今、アクセルを踏みしめて揺るがないこの覚悟は、もっと大切なもののためだった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "今、アクセルを踏みしめて揺るがないこの覚悟は、もっと大切なもののためだった。",

> 			NULL, "Right now, the unshaking resolve she had to step down on the accelerator was for something more important.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Right now, the unshaking resolve pushing her to to step down on the accelerator was for something more important.", Line_Normal);
```
